vector/src/main/java/im/vector/app/features/location/LocationSharingService.kt

```diff
@@ -67,11 +78,40 @@ class LocationSharingService : VectorService(), LocationTracker.Callback {
 
             // Schedule a timer to stop sharing
             scheduleTimer(roomArgs.roomId, roomArgs.durationMillis)
+
+            // Send beacon info state event
+            activeSessionHolder
+                    .getSafeActiveSession()
+                    ?.let { session ->
+                        session.coroutineScope.launch {
+                            sendBeaconInfo(session, roomArgs)
+                        }
+                    }
```

Comment:
Just to double check: this code does not send multiple beacon events during a period of time, but just one event, right?

Reply:
Yes, location events will be sent later.